src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -184,6 +211,58 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
     this.inlineFormClose.emit(inlineFormCloseArgs);
   }
 
+  public moveToTop(event: Event): void {
+    event.stopPropagation();
+    this.adapterService.moveItemUp(this.elementRef, true);
+  }
+
+  public handleKeyboardEvent(event: KeyboardEvent): void {
+    let key = event.key.toLowerCase();
+    if (key === ' ') {
+      this.keyboardDraggingEnabled = !this.keyboardDraggingEnabled;
+      this.reorderSteps = 0;
+
+      if (this.keyboardDraggingEnabled) {
+        this.reorderState = this.reorderStateDescription;
+      } else {
+        this.reorderState = undefined;
+      }
+
+      event.stopPropagation();
+    } else if (key === 'esc') {
+      this.keyboardDraggingEnabled = false;
+      this.reorderState = undefined;
+
+      if (this.reorderSteps < 0) {
+        this.adapterService.moveItemDown(this.elementRef, Math.abs(this.reorderSteps));
+      } else if (this.reorderSteps > 0) {
+        this.adapterService.moveItemUp(this.elementRef, false, this.reorderSteps);
+      }
+
+      event.stopPropagation();
+    } else if (this.keyboardDraggingEnabled && key.startsWith('arrow')) {
+      let direction = event.key.toLowerCase().replace('arrow', '');
+      if (direction === 'up') {
+        this.adapterService.moveItemUp(this.elementRef);
+        this.reorderSteps--;
+        this.grabHandle.nativeElement.focus();
+        this.keyboardDraggingEnabled = true;
+      } else if (direction === 'down') {
+        this.adapterService.moveItemDown(this.elementRef);
+        this.reorderSteps++;
+        this.grabHandle.nativeElement.focus();
+        this.keyboardDraggingEnabled = true;
+      }
+
+      event.stopPropagation();
```

Comment:
The up and down keys scroll the browser window up and down when dragging is activated. Should we also `preventDefault()`?

Reply:
Awesome catch, I'd looked at it multiple times and hadn't noticed that. I've added that call and can confirm that it stops the scrolling.